client/my-sites/plugins-wpcom/jetpack-plugin-item.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { identity } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Button from 'components/button';
+import PurchaseButton from './purchase-button';
+
+export const JetpackPluginItem = ( {
+	plugin,
+	siteSlug,
+	isActive,
+	translate = identity
+} ) => {
+	let plan;
+
+	if ( plugin.plan === 'premium' ) {
+		plan = <Button compact borderless className="plugins-wpcom__is-premium-plugin">
+			{ translate( 'Premium' ) }
+		</Button>;
+	} else if ( plugin.plan === 'business' ) {
+		plan = <Button compact borderless className="plugins-wpcom__is-business-plugin">
+			{ translate( 'Business' ) }
+		</Button>;
+	} else {
+		plan = null;
+	}
```

Comment:
We can save some DRY here by doing something like this:

```
const translatePlan = {
    premium: translate( 'Premium' ),
    business: translate( 'Business' )
};

const plan = [ 'premium', 'business' ].includes( plugin.plan ) ? 
    <Button compact borderless className=`plugins-wpcom__is-${ plugin.plan }-plugin`>{ translatePlan[ plugin.plan ] }</Button> :
    null;
```

But this could perhaps be more difficult to read in the end. But we save some lines.